CommandLineOptions.cpp

```diff
@@ -55,10 +56,25 @@ void SetPlaygroundOption(std::string opt, std::string val, CommandLineOptions &o
     opts.is_playground = is_playground == 1;
 }
 
+void HandleOptionWithoutEquals(std::string arg, CommandLineOptions &opts)
+{
+    // NOTE(sasha): We did ToLowerCase earlier, so -I turns into -i and -L turns into -l
+    // NOTE(sasha): The 2 comes from the length of "-i" or "-l"
+    if(StartsWith(arg, "-i"))
```

Comment:
This really is not valid - `-isystem` beings with `-i` but has a different behaviour than `-I`.